stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -1897,13 +1867,7 @@ extension Data {
 
 /// Provides bridging functionality for struct Data to class NSData and vice-versa.
 
-#if DEPLOYMENT_RUNTIME_SWIFT
```

Comment:
Is this change orthogonal to the copyContents one?

Reply:
Correct but it allows me to iterate on sclf for validating this easily